### DISCUSSION TITLE
Peptide lenght and haplotype considered by pVAC bind

### DISCUSSION THREAD

**Discussion by martinez14recerca** [griffithlab/pVACtools#1102] [Q&A] [answered]
Hello,
I have recently discovered this pipeline for neoantigen characterization. I would like to know if, when running through the different programs used by pVAC bind, if I am providing a neoantigen with a different length one of the programs does not recognise or if the HLA haplotype is not considered in one of the databases of those programs, if the pipeline running will be terminated or if it will just skip this program and move to the next one.
Thank you very much.

**Comment by chrisamiller** [comment, marked as answer]
pVACtools will skip incompatible HLA types/lengths and return results for which it can produce results. Hope this helps!
